lib/cesiumGeometryToGltfPrimitive.js

```diff
@@ -1,10 +1,74 @@
 'use strict';
+var Cesium = require('cesium');
+var findAccessorMinMax = require('./findAccessorMinMax');
+var getUniqueId = require('./getUniqueId');
+var mergeBuffers = require('./mergeBuffers');
+var numberOfComponentsForType = require('./numberOfComponentsForType');
+var uninterleaveAndPackBuffers = require('./uninterleaveAndPackBuffers');
 var writeAccessor = require('./writeAccessor');
 
+var WebGLConstants = Cesium.WebGLConstants;
+
 module.exports = cesiumGeometryToGltfPrimitive;
 
+// Creates a buffer location to write the new values to. mapGeometryAttributeToPrimitive does the actual writing
+function createAttributeSemantic(gltf, primitive, semantic, packedLength) {
+    var type;
+    switch (semantic.replace(/([0-9]|_)/g, '')) {
```

Comment:
I don't have a strong preference either way, your suggestion is more compact and a little clearer, so I say go with that.